full_automation.py

```diff
@@ -11,9 +11,57 @@
 from utils.constants import model2base_model, model2size
 from utils.flock_api import get_task, submit_task
 
+import sys
+from git import Repo, GitCommandError
+
 HF_USERNAME = os.environ["HF_USERNAME"]
 
+def check_and_update_repo():
```

Comment:
I think we can mimic what's been done in https://github.com/FLock-io/llm-loss-validator/pull/26
We probably shouldn't automatically pull the newest main for user. They might have done some hack on their local

Reply:
yes, I delete the auto pull newest version, and just show some warning.